setup.py

```diff
@@ -31,20 +32,6 @@
         'Natural Language :: English',
         'Operating System :: OS Independent'],
     python_requires='>=3.6',
```

Comment:
The CI test fails, I wonder if it's not installing properly with these removed?

Reply:
I think I forgot to add emcpy to the gh requirements